docs/product-administration-and-configuration-guide/src/main/asciidoc/configuring-openshift-connection-timeout-proc.adoc

```diff
@@ -0,0 +1,30 @@
+[id='configuring-openshift-connection-timeout-proc']
+= Configuring OpenShift connection timeout
+
+By default, OpenShift route is configured to timeout HTTP requests that are longer than 30 seconds. This may cause session timeout issues in {CENTRAL} resulting in the following behaviors:
+
+* "Unable to complete your request. The following exception occurred: (TypeError) : Cannot read property 'indexOf' of null."
+* "Unable to complete your request. The following exception occurred: (TypeError) : b is null."
+* A blank page is displayed when clicking the *Project* or *Server* links in {CENTRAL}.
+
```

Comment:
Maybe include a observation that all of our templates already have it. configured